raspi/start-robot.sh

```diff
@@ -0,0 +1,9 @@
+:
+cd ~/pigmice-vision
+
+if ifconfig | grep 10.27.33.20:
+then
+  python robot_vision.py
```

Comment:
Since this program will be running *headless*, we could save any debugging values if we change this line to:

      python robot_vision.py 2>&1. > /home/pi/robot_vision_output.txt

Then, if we ever want to see what it is printing, we could ssh into the pi, and `cat` this file.